frontend/catalyst/utils/runtime.py

```diff
@@ -239,6 +240,14 @@ def extract_backend_info(device):
     device_lpath = ""
     device_kwargs = {}
 
+    with open(device.config, "rb") as f:
+        config = toml_load(f)
+
+    if config["compilation"].get("jax_transform_compatible"):
```

Comment:
Maybe this name is a bit generic, or do you mean that a device would in general declare that it is compatible with jax transforms (I would think it should be oblivious to that)?

Reply:
Same concern.

Reply:
`qjit_compatible` to me means that a device will be interfaced via the shared library. This CUDA quantum device is not implemented that way. So `qjit_compatible` should be set to false. We need a way to communicate that this is a different device that will not go through the normal pipeline. Do you have any suggestions for an alternative flag?

Reply:
Doesn't `qjit_compatible` simply mean it can be compiled with `qjit`? I think that was the original intention, because `lightning.gpu` for example was not qjit compatible.

Reply:
As for the CUDA compiler devices, maybe they shouldn't use this function at all if it is Catalyst specific? 🤔

Reply:
> Doesn't qjit_compatible simply mean it can be compiled with qjit? I think that was the original intention, because lightning.gpu for example was not qjit compatible.

Yep that was the original intention

Reply:
I made the changes here: https://github.com/PennyLaneAI/catalyst/pull/477/commits/940654497a73a2a86b35d1814d385f1a97dd1df1